src/main/java/api/clients/ApiLabelClient.java

```diff
@@ -0,0 +1,41 @@
+package api.clients;
+
+import api.BaseRestTestClient;
+import api.pojo.responses.LabelResponse;
+import io.qameta.allure.Step;
+import io.restassured.response.Response;
+
+import static io.restassured.RestAssured.given;
+
+public class ApiLabelClient extends BaseRestTestClient {
+    public ApiLabelClient(String url) {
+        super(url);
+    }
+
+    @Step("Create new label. Expected status code {expectedStatusCode}")
```

Comment:
Add in description of step `labelName`,`labelColor`,`idBoard` and `expectedStatusCode` parameters